### PR TITLE
Add AMD GPU node labeller to scheduling-gpus.md

### PR DESCRIPTION
**Pull request by y2kenny**

### CODE REVIEW

File: content/en/docs/tasks/manage-gpus/scheduling-gpus.md [hunk 1]
```diff
@@ -156,6 +156,39 @@ kubectl label nodes <node-with-k80> accelerator=nvidia-tesla-k80
...
 ```
 
+For AMD GPUs, you can deploy [Node Labeller](https://github.com/RadeonOpenCompute/k8s-device-plugin/tree/master/cmd/k8s-node-labeller), which automatically label your nodes with GPU properties.  Currently supported properties are:
+
+* Device ID (-device-id)
+* VRAM Size (-vram)
+* Number of SIMD (-simd-count)
+* Number of Compute Unit (-cu-count)
+* Firmware and Feature Verions
+* GPU Family, in two letters acronym (-family)
```
Comment (kbhawkey) [inline content/en/docs/tasks/manage-gpus/scheduling-gpus.md]: nit: Is it Firmware and Feature Versions?

Comment (y2kenny) [reply content/en/docs/tasks/manage-gpus/scheduling-gpus.md]: Yes.  It is both.  But I noticed I missed a flag there.

Comment (kbhawkey) [reply content/en/docs/tasks/manage-gpus/scheduling-gpus.md]: spelling: Verions?

Comment (y2kenny) [reply content/en/docs/tasks/manage-gpus/scheduling-gpus.md]: Oh right... sorry I missed that.

**Comment by Bradamant3** [comment]
/lgtm
(re-adding post edit commit)
/approve

**Pull request merged**